crates/sui-types/src/error.rs

```diff
@@ -98,9 +98,10 @@ pub enum UserInputError {
         object_id: ObjectID,
         version: Option<SequenceNumber>,
     },
-    #[error("Object {provided_obj_ref:?} is not available for consumption, its current version: {current_version:?}")]
+    #[error("Object {object_id} with version {object_version} is not available for consumption, current version: {current_version}")]
     ObjectVersionUnavailableForConsumption {
-        provided_obj_ref: ObjectRef,
+        object_id: ObjectID,
+        object_version: SequenceNumber,
```

Comment:
Pulled up the digest into the error type in my today, continuing on the path I'm on.

The other options I considered:
- pull the tuple elements up a level, which is shown here
- impl display, requires changing from a type definition to a struct definition, possible but requires maybe twice as much code to change all the places where the type is instantiated
- change the error display inside of the impl From<Error>, very messy because its nested 2 levels deep within the `QuorumDriverError`
- Could just keep the debug error but doesn't look very great.